interface/backend/api/urls.py

```diff
@@ -1,4 +1,4 @@
-from collections import OrderedDict
+import collections
```

Comment:
Why do you prefer to import the whole module here while you changed in 5302cfe401a1f77fa878fd8ad1bc26dd2d77e224 to directly import methods from a module?

Reply:
So in general I prefer to import the modules that Python ships with ("batteries included") as top-level modules. However, I am not 100% dogmatic on this (!) and make some exceptions if that's going to make the code unspeakably ugly. For example,, imagine trying to parse:
```
os.path.join(os.path.join(os.path.join(os.path.ditpath(os.path.join(os.path.join(os.path.join(....)))
```
Make sense? :)

Reply:
Yes, thank you :)